requirements.txt

```diff
@@ -1,4 +1,5 @@
-Flask==2.0.2
+Flask==2.2.3
+requests==2.29.0
```

Comment:
<!-- Polish comments are like a suggestion, where there is nothing necessarily wrong with the relevant content, there's just some ways to immediately improve the quality. -->
**polish:** Maybe a comment here to remind us why it is pinned, so we can remove it one day.